docs/src/prototypes/popups/GridImagePicker/GridImagePickerItem.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react'
+import { Image, Button } from '@stardust-ui/react'
+import * as _ from 'lodash'
+
+export interface GridPickerItemProps {
+  as?: keyof React.ReactHTML
+  title?: string
+  imageSrc: string
+  onClick?: (e) => void
```

Comment:
`e` should have type definition,

```suggestion
  onClick?: (e: React.SyntethicEvent) => void
```

Reply:
It also doesn't match definition, the correct signature is:

```suggestion
  onClick?: (e: React.SyntethicEvent, props: GridPickerItemProps) => void
```